chipy_org/apps/main/views.py

```diff
@@ -36,10 +29,7 @@ def get_context_data(self, **kwargs):
         context = {}
         context.update(kwargs)
         context["IS_HOMEPAGE"] = True
-        context["other_meetings"] = self.get_non_main_meetings(num=3)
         context["featured_sponsor"] = Sponsor.featured_sponsor()
-        context["sponsor_groups"] = SponsorGroup.objects.prefetch_related("sponsors")
```

Comment:
Were these removed intentionally? They seem unrelated to the __main__ vs other meeting fix 🤔

Reply:
Yes, because they're no longer needed in the redesigned homepage.